src/NHSD.BuyingCatalogue.Organisations.Api/Models/ApplicationUser.cs

```diff
@@ -45,13 +44,12 @@ private ApplicationUser()
             if (email is null)
                 throw new ArgumentNullException(nameof(email));
 
-            Id = userId.ToString();
-            UserName = userName?.Trim();
-            NormalizedUserName = UserName?.ToUpperInvariant();
-            FirstName = firstName?.Trim();
-            LastName = lastName?.Trim();
+            UserName = userName.Trim();
+            NormalizedUserName = UserName.ToUpperInvariant();
+            FirstName = firstName.Trim();
+            LastName = lastName.Trim();
             PhoneNumber = phoneNumber ?? throw new ArgumentNullException(nameof(phoneNumber));
-            Email = email?.Trim();
+            Email = email.Trim();
             NormalizedEmail = Email?.ToUpperInvariant();
```

Comment:
As above, would it make sense to use the parameter consistently here? Either way, we don't need the conditional null operator anymore.

Reply:
👍  On the null operator.